README.md

```diff
@@ -1,3 +1,8 @@
 # DSCI_532_group_205
-Movies Dashboard in Python
-Remote change from sree
\ No newline at end of file
+Movies Dashboard in Python  
+
+
+## Description of the app
+
+## Sketch of the app
+!["sketch"](img/sketch.jpg)
```

Comment:
I think the Genre panel can have check boxes and MPAA Rating panel can have radio buttons to make the interactivity more explicit

Reply:
Check boxes for rating is fine too. I might want to look for NC-17 and R films at the same time.

Reply:
But can we select multiple options with radio buttons?

Reply:
Agree with Ryan

Reply:
@fkhan72 @sreejithmunthikodu Do you agree?

Reply:
sounds good